src/tests/components.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-expressions */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Home from '../components/home';
+import Quote from '../components/qoute';
+import Calculator from '../components/Calculator';
+
+describe('tests for components', () => {
+  test('Home should be in the document', () => {
+    render(<Home />);
+    const home = screen.getByRole('heading');
+    expect(home)
+      .toBeInTheDocument; /* eslint-disable-line no-unused-expressions */
+  });
+  test('Quote should be in the document', () => {
+    render(<Quote />);
+    const quote = screen.getByPlaceholderText('Quote');
+    expect(quote)
+      .toBeInTheDocument; /* eslint-disable-line no-unused-expressions */
+  });
+  test('Calculator should render in the document', () => {
+    render(<Calculator />);
+    // const calculator = screen.getByRole('heading');
+    const calculator = () => {
+      screen.getByRole('heading');
+    };
+    expect(calculator)
+      .toBeInTheDocument; /* eslint-disable-line no-unused-expressions */
+  });
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20Jest%20snapshots%20to%20test%20the%20components.), Please use [jest snapshot testing ](https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest)for testing your components instead of what you did here.

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20React%20Testing%20Library%20to%20simulate%20user%20interaction.), Please add tests to simulate user interaction using React Testing Library.